packages/esm-sms-app/src/providers/providers-overview/providers-action-menu.component.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useCallback, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Layer, OverflowMenu, OverflowMenuItem } from '@carbon/react';
+import { useLayoutType, showModal, ExtensionSlot, showSnackbar, launchWorkspace } from '@openmrs/esm-framework';
+import styles from './providers-action-menu.scss';
+import { type ProviderConfiguration } from '../../types';
+import { useProviderConfigurations } from '../../hooks/useProviderConfigurations';
+import { setAsDefaultConfig } from '../../api/providers.resource';
+
+interface configurationsActionMenuProps {
+  config: ProviderConfiguration;
+}
+
+export const ConfigurationsActionMenu = ({ config }: configurationsActionMenuProps) => {
+  const { defaultConfig, mutateConfigs, providerConfigurations } = useProviderConfigurations();
+  const { t } = useTranslation();
+  const isTablet = useLayoutType() === 'tablet';
+  const isDefaultConfig = config.name === defaultConfig;
+  const editConfigWorkspaceTitle = config
+    ? t(`edit${config.name}`, `Edit ${config.name}`)
```

Comment:
This is not how i18next [interpolation](https://www.i18next.com/translation-function/interpolation) works.